lr_face/utils.py

```diff
@@ -171,3 +188,15 @@ def concat_columns(df, column_names, output_column_name, separator='-'):
     for i in range(1, len(column_names)):
         df[output_column_name] += separator + df[column_names[i]].astype(str)
     return df
+
+
+def resize(img, target_size):
+        right_size_img  = cv2.resize(img, target_size)
+
+        img_pixels = image.img_to_array(right_size_img)
+        img_pixels = np.expand_dims(img_pixels, axis=0)
+
+        # normalize input in [0, 1]
+        img_pixels /= 255
```

Comment:
waarom is normalizen nodig in een resize functie? klinkt als een aparte functie, of een resize_and_normalize functie?

Reply:
Dit is nog een leftover van de enorme decect/crop/align/resize functie uit deelface, ik heb alleen het laatste stukje daarvan genomen. Zal hem hernoemen.